test/parallel/test-cluster-server-restart-none.js

```diff
@@ -10,15 +10,15 @@ if (cluster.isMaster) {
   worker1.on('listening', common.mustCall(() => {
     const worker2 = cluster.fork();
     worker2.on('exit', (code, signal) => {
-      assert.strictEqual(code, 0, 'worker2 did not exit normally');
-      assert.strictEqual(signal, null, 'worker2 did not exit normally');
+      assert.strictEqual(code, 0, `worker2 did not exit normally. exited with code ${code}`);
+      assert.strictEqual(signal, null, `worker2 did not exit normally. exited with signal ${signal}`);
       worker1.disconnect();
     });
   }));
 
   worker1.on('exit', common.mustCall((code, signal) => {
-    assert.strictEqual(code, 0, 'worker1 did not exit normally');
-    assert.strictEqual(signal, null, 'worker1 did not exit normally');
+    assert.strictEqual(code, 0, `worker1 did not exit normally. exited with code ${code}`);
+    assert.strictEqual(signal, null, `worker1 did not exit normally. exited with signal ${signal}`);
```

Comment:
The line length exceeds 80 characters here. This has to be fixed but it can also be done while landing.

Reply:
done